### PR TITLE
Add docs for service create based on plugins.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Signed-off-by: Anusha Ragunathan <anusha.ragunathan@docker.com

### CODE REVIEW

**Comment by anusha-ragunathan** [comment]
@aaronlehmann @stevvooe PTAL. This primarily documents working of plugin filters in 1.13

File: docs/extend/index.md [hunk 1]
```diff
@@ -107,6 +107,62 @@ remove it, use the `docker plugin remove` command. For other available
...
+    $ docker service ls
+    z1sj8bb8jnfn  my-service   replicated  1/1       busybox:latest 
+    ```
+    docker service ls shows service 1 instance of service running.
+
+4. Observe the task getting scheduled in node 2:
+
+    ```bash
+    $ docker ps --format '{{.ID}}\t {{.Status}} {{.Names}} {{.Command}}' 
+    83fc1e842599     Up 2 days my-service.1.9jn59qzn7nbc3m0zt1hij12xs "top"
```
Comment (stevvooe) [inline docs/extend/index.md]: This output looks different that the format. Is there time in the `.Status` field?

Comment (anusha-ragunathan) [inline docs/extend/index.md]: Not absolute time, only uptime

**Comment by stevvooe** [comment]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸
/cc @thaJeztah

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**